content/en/docs/getting-started/_index.md

```diff
@@ -220,3 +220,29 @@ oc.exe get all -A --kubeconfig .\kubeconfig
 ```
 {{% /tab %}}
 {{< /tabs >}}
+
+## MicroShift on OSTree based systems
+
+As mentioned aboved, MicroShift has been designed to be deployed on edge computing devices. Looking at security standards, 
+an edge optimized operating system will most likely be inmutable and based in transactions for upgrades and rollbacks. OSTree provides these capabilities. 
+
+Fedora IoT and RHEL for Edge are both OSTree based systems and MicroShift can be shipped as part of the base `rpm-ostree`. 
+The recommended way to embed MicroShift in these operating systems is to build your own `rpm-ostree` with tools like [Image Builder](https://fedoramagazine.org/introduction-to-image-builder/). This project will allow you to create your own customized version of Fedora IoT or RHEL for Edge in order to include MicroShift and all the required dependencies like CRI-O.
+
+However, developers might need to manually install RPMs on the system for faster iterations. It is important to highlight that the base layer of an `rpm-ostree` is an atomic entity, so when installing a local package, any dependency that is part of the ostree with an older version will not be updated. This is the reason why it is mandatory to perform an upgrade before manually installing MicroShift.
+
+
+Let's see an example of a Fedora IoT 35 system:
+
+```
+curl -L -o /etc/yum.repos.d/fedora-modular.repo https://src.fedoraproject.org/rpms/fedora-repos/raw/rawhide/f/fedora-modular.repo
+curl -L -o /etc/yum.repos.d/fedora-updates-modular.repo https://src.fedoraproject.org/rpms/fedora-repos/raw/rawhide/f/fedora-updates-modular.repo
+curl -L -o /etc/yum.repos.d/group_redhat-et-microshift-fedora-35.repo https://copr.fedorainfracloud.org/coprs/g/redhat-et/microshift/repo/fedora-35/group_redhat-et-microshift-fedora-35.repo
+rpm-ostree ex module enable cri-o:1.21
+
+rpm-ostree upgrade
+systemctl reboot
+
+rpm-ostree install cri-o cri-tools microshift
```

Comment:
```suggestion
rpm-ostree install cri-o cri-tools microshift
systemctl reboot
```
? I havent tried, but wondering if we need it.

Reply:
Do you mean if we need 2 reboots?

Reply:
we only need one reboot. You can upgrade + install microshift without an intermidiate reboot. Thanks for the catch!